datacenter/ucp/2.2/guides/release-notes.md

```diff
@@ -10,6 +10,60 @@ known issues for the latest UCP version.
 You can then use [the upgrade instructions](admin/install/upgrade.md), to
 upgrade your installation to the latest release.
 
+## Version 2.2.6 (2018-03-19)
```

Comment:
in past releases, we put the date in a separate line, can you change it for consistency?

Reply:
UCP and DTR were the exception rather than the rule. All other release notes were using this format, so let's standardize on that.

I've updated the release notes for 2.2.x to start using that format.